crypto/provider.c

```diff
@@ -13,6 +13,24 @@
 #include <openssl/core_names.h>
 #include "internal/provider.h"
 
+OSSL_PROVIDER *OSSL_PROVIDER_load_leave_fallbacks(OSSL_LIB_CTX *libctx,
```

Comment:
This should just be OSSL_PROVIDER_try_load which does this almost identical logic. 
We should not be adding a new function like this.

Now OSSL_PROVIDER_try_load seems poorly named - but that is another issue - and adding a replacement identical function isn't the right solution.

Reply:
It can't be OSSL_PROVIDER_try_load() because that removes the possibility of using the fallbacks if the provider successfully loads.  This capability is required for the tests.

Reply:
Switched this to use an argument on OSSL_PROVIDER_try_load() instead.